racket/collects/compiler/cm.rkt

```diff
@@ -15,6 +15,9 @@
          make-compile-lock
          compile-lock->parallel-lock-client
 
+         ;; re-used in setup/setup-core and setup/private/pkg-deps
+         cross-multi-compile?
```

Comment:
Probably this function should be put into its own module, so the pubic interface of `compiler/cm` doesn't grow, and so it can be used without importing all the rest of `compiler/cm`.